aws/ec2.go

```diff
@@ -0,0 +1,84 @@
+package aws
+
+import (
+	"fmt"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/aws-nuke/logging"
+)
+
+var ec2Instances = make(map[string][]*string)
+
+// Build string that'll be shown in resources list
+func buildEntryName(instance ec2.Instance) string {
+	return fmt.Sprintf("ec2-%s-%s", *instance.InstanceId, *instance.InstanceType)
```

Comment:
There is an `aws.StringValue` function that is a `nil`-safe way to turn a pointer into a string.